stix_shifter_modules/qradar/stix_translation/json/to_stix_map.json

```diff
@@ -273,12 +284,21 @@
   },
   "credibility": {
     "key": "x_com_ibm_ariel.credibility",
-    "cybox": false
-  },
-  "severity": {
-    "key": "x_com_ibm_ariel.severity",
-    "cybox": false
+    "cybox": false,
+    "transformer": "ToInteger"
   },
+  "severity": [
+    {
+      "key": "x_com_ibm_ariel.severity",
+      "cybox": false,
+      "transformer": "ToInteger"
+    },
+    {
+      "key": "x_risk_finding.severity",
+      "cybox": false,
+      "transformer": "ToInteger"
```

Comment:
if it's not a requirements for insider threat than we can just keep the severity in one place may be? just trying to avoid redundant data.